IFC4x3/Properties/w/WindSpeedRange_3prEZg_014axLq1m30EpZt/Documentation.md

```diff
@@ -0,0 +1 @@
+Indicate the range of wind speed the anemometrograph can monitor.
```

Comment:
Generalize?

Reply:
Updated the documentation but still keep it as a speed range only for wind. https://github.com/bSI-InfraRoom/IFC-Specification/pull/165/commits/1e8b0740cc2e7e000d58ef2f240b7e5a49640ee9